articles/stream-analytics/stream-analytics-define-inputs.md

```diff
@@ -98,6 +105,7 @@ The following table explains each property in the **New input** blade in the Azu
 | **Consumer group** (optional) |The consumer group to use to ingest data from the IoT hub. If no consumer group is specified, a Stream Analytics job uses the default consumer group. We recommend that you use a different consumer group for each Stream Analytics job. |
 | **Event serialization format** |The serialization format (JSON, CSV, or Avro) of the incoming data stream. |
 | **Encoding** |UTF-8 is currently the only supported encoding format. |
+| **Compression** (optional) | The compression type (None, GZip, or Deflate) of the incoming data stream. |
```

Comment:
Since this compression setting is optional, please call out what is the default value.